### PR TITLE
Add moreinfo summary

### PR DESCRIPTION
**Pull request by cthorey**
1. Made the CERES summary available int he calibration node
2. Dump several info from the summary along the calibration transform in CalibratedTransform.yml

### CODE REVIEW

File: src/handeye_calibration.cpp [hunk 1]
```diff
@@ -82,41 +85,41 @@ int readTransformPairsFromFile(std::string filename, EigenAffineVector& t1, Eige
...
-				fs[ss2.str()] >> t2cv;
-				Eigen::Affine3d t2e;
-				cv::cv2eigen(t2cv,t2e.matrix());
-				*t2_it = t2e;
-			}
-
-		}
-		fs.release();
-	} else {
+    {
```
Comment (ahundt) [inline src/handeye_calibration.cpp]: some of these indentation changes seem a little weird.

File: src/handeye_calibration.cpp [hunk 2]
```diff
@@ -82,41 +85,41 @@ int readTransformPairsFromFile(std::string filename, EigenAffineVector& t1, Eige
...
+      fs.release();
+    } else {
 		std::cerr << "failed to open input file " << filename << "\n";
 		return 1;
 	}
 	return 0;
 }
 
-Eigen::Affine3d estimateHandEye(const EigenAffineVector& baseToTip, const EigenAffineVector& camToTag)
+Eigen::Affine3d estimateHandEye(const EigenAffineVector& baseToTip, const EigenAffineVector& camToTag, ceres::Solver::Summary& summary)
```
Comment (ahundt) [inline src/handeye_calibration.cpp]: I think the summary info is great to have, but can the summary be an optional parameter?

I'd like to keep this function as easy to use as possible in the most basic case.

Comment (cthorey) [reply src/handeye_calibration.cpp]: I'll try to look into how to do that in C++ over the wee-end.

Comment (ahundt) [reply src/handeye_calibration.cpp]: I think just adding an overload (another function definition) without it that creates an unused one internally and then calls this version would be fine.

File: example/CalibratedTransform.yml [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
+       6.5803887979601350e-02, 7.8462443596179002e-03,
+       4.9926199062053020e-02, -8.9600998786831365e-02,
+       9.9472560822752454e-01, 8.3531588496032982e-02,
+       1.8900684554033920e-01, 9.7881847125794519e-01,
+       7.8681717464412104e-02, -8.7184693633301849e-03, 0., 0., 0., 1. ]
+initial_cost: 9.6952771603052695e-04
+final_cost: 6.0056906684146673e-04
+change_cost: 3.6895864918906022e-04
+termination_type: 0
+num_succefull_iteration: 73
```
Comment (ahundt) [inline example/CalibratedTransform.yml]: successful spelling error, also on the next line

**Comment by cthorey** [comment]
See PR #14

**Pull request closed without merging**